src/Command/CleanerCommand.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace SymfonyCasts\Bundle\ResetPassword\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use SymfonyCasts\Bundle\ResetPassword\Util\ResetPasswordCleaner;
+
+/**
+ * @author  Jesse Rushlow <jr@rushlow.dev>
+ * @author Ryan Weaver <ryan@symfonycasts.com>
+ *
+ * @final
+ */
+class CleanerCommand extends Command
```

Comment:
ResetPasswordRemoveExpiredCommand - it's always nice to name it close to the command name... it helps when I try to search for a command class lazily based on its command string :)

Reply:
done